### DISCUSSION TITLE
[Bug]: documents_editables is not updated in Pimcore 6 to X update

### DISCUSSION THREAD

**Discussion by Cruiser13** [pimcore/pimcore#13647] [Q&A / Support] [answered]
### Expected behavior

Updating Pimcore 6 to X should run migrations that'll ensure content on documents does still exist.

### Actual behavior

Our Pimcore 6 instance editables are not updated during Pimcore X upgrade. All content ist lost.

Example for a pimcore areablock named "firstarea" with an image inside.

documents_editables looks like this:
**name: imagefirstarea7**
type: image
data: a:9:{s:2:"id";i:2477;s:3:"alt";s:0:"";s:11:"cropPercent";N;s:9:"cropWidth";N;s:10:"cropHeight";N;s:7:"cropTop";N;s:8:"cropLeft";N;s:8:"hotspots";a:0:{}s:6:"marker";a:0:{}}

while it should look like this:
**name: firstarea:1.image**
type: image
data: a:10:{s:2:"id";i:187;s:3:"alt";s:0:"";s:11:"cropPercent";N;s:9:"cropWidth";N;s:10:"cropHeight";N;s:7:"cropTop";N;s:8:"cropLeft";N;s:8:"hotspots";N;s:6:"marker";N;s:9:"thumbnail";s:7:"content";}

While the editable structure is still in place (areablock contains an image, followed by wysiwyg) all content is no longer shown. Content is still present in the database

### Steps to reproduce

We can not provide a full demo setup here. The Pimcore instance has been migrated from Pimcore 4 to 5 to 6. We switched from PHPTemplates to Twig Templates during the pimcore X process.

Any pointer to solve this issue would be very welcome!

**Comment by brusch** [comment, marked as answer]
You need to migrate to the new naming strategy before upgrading to Pimcore X, see: 
https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Documents/Editable_Naming_Strategies.html

**Reply by Cruiser13** [reply]
Thank you @brusch. Pimcore 5 has been so long ago, that was no longer on my mind. Interesting that this has not been done by us during the 6 upgrade.
